content/2015.10.kinto-js-realeased.rst

```diff
@@ -0,0 +1,83 @@
+
+Kinto.js provides basic local CRUD operations for `creating <https://kintojs.readthedocs.org/en/latest/api/#creating-a-record>`_, `updating <https://kintojs.readthedocs.org/en/latest/api/#updating-a-record>`_, `deleting <https://kintojs.readthedocs.org/en/latest/api/#deleting-records>`_, `retrieving <https://kintojs.readthedocs.org/en/latest/api/#retrieving-a-single-record>`_ and `listing <https://kintojs.readthedocs.org/en/latest/api/#listing-records>`_ records from a local collection. Each time you want to reflect its state to a remote server, you only ever have to `call the sync() method <https://kintojs.readthedocs.org/en/latest/api/#fetching-and-publishing-changes>`_!
+
+We've set up a `simple demo application <https://kinto.github.io/kinto.js/>`_ using a public Mozilla test server instance, so you can check how it behaves by yourself if you're interested. The source code of this app is `available on Github <https://github.com/Kinto/kinto.js/blob/66ef077796a8e4a8b0d9a95389a61ea57cbbfde3/demo/demo.js>`_, and is also the base use case for the `Kinto.js tutorial <http://kintojs.readthedocs.org/en/latest/tutorial/>`_ you should probably take if you want to get your feet wet.
+
+.. note::
+
+   Please note that this public Kinto test server instance is flushed every morning at 7AM UTC, so don't rely on it for production use ;)
+
+Synchronizing and reconciling data
+----------------------------------
+
+Synchronizing local and remote data is a tough topic, and conflicts may occur quite easily if things are changing often from different sources (think multiple devices updating a same server); Kinto.js supports three distinct *synchronization strategies* to handle such a situation:
+
+- The server is authoritative: all conflicting records are overriden locally by their version from the server;
+- The client is authoritative: all conflicting records are resolved using their local versions, and force-pushed to the server;
+- No party is authoritative: the synchronization flow is halted, and all conflicting records are explicitely exposed, so developers can then decide how they want resolve each of them individually.
+
+You can read more about `conflict resolution <https://kintojs.readthedocs.org/en/latest/api/#resolving-conflicts-manually>`_ and `sychronization strategies <http://kintojs.readthedocs.org/en/latest/api/#synchronization-strategies>`_ in the dedicated sections of the project documentation.
+
+The diagram flow below describes pretty much accurately what's happening under the hood when ``sync()`` is called:
+
+.. image:: {filename}/images/kintojs-sync-flow.png
+   :alt: Kinto.js synchronization flow
+
+A case for encryption
+---------------------
+
+Remote data storage is certainly a concerns-raising topic when it comes to security and privacy.
```

Comment:
s/concerns-raising topic/topic raising concerns/   (i don't think concerns-raising is an adjective in english)